engine/src/main/resources/templates/ImageBuilder/launchFailedInstantEmailTitleV2.txt

```diff
@@ -0,0 +1 @@
+Instant notification - image launch failed - Red Hat Enterprise Linux
```

Comment:
Should be "Instant notification - Image launch failed - Image builder - Red Hat Enterprise Linux"